src/routes/blog/index.svelte

```diff
@@ -1,9 +1,25 @@
 <script lang="ts" context="module">
   import type { LoadInput, LoadOutput } from '@sveltejs/kit/types/internal';
+  import TagField from '$lib/components/utils/acm-tagfield.svelte';
 
-  export async function load({ fetch }: LoadInput): Promise<LoadOutput> {
-    const response = await fetch(`/blog.json`);
-    return { props: { posts: await response.json() } };
+  let tags = [];
+  let selectedTags: string[] = [];
```

Comment:
These two variables should not be reachable from the `module` script. The `module` script is ran on the server and sometimes on the browser. Therefore, we want to remove the variable declarations here and move them down to the 2nd `<script>`.
```suggestion
```

Reply:
I put those two variables in `module` script on purpose. Their values will be set in the `load()` function.

Reply:
Yup, I will fix it. Now I feel like I begin to understand Svelte.